Actions/BuildReferenceDocumentation/BuildReferenceDocumentation.HelperFunctions.ps1

```diff
@@ -24,7 +24,7 @@
             & /usr/bin/env sudo pwsh -command "& chmod +x $ENV:aldocPath"
         }
         Write-Host "Installing/Updating docfx"
-        CmdDo -command dotnet -arguments @("tool","update","-g docfx") -messageIfCmdNotFound "dotnet not found. Please install it from https://dotnet.microsoft.com/download"
+        CmdDo -command dotnet -arguments @("tool","update","--global docfx --version 2.75.3") -messageIfCmdNotFound "dotnet not found. Please install it from https://dotnet.microsoft.com/download"
```

Comment:
Why the hardcoded version here?

Reply:
Because docfx 2.76 released and broke aldoc

Reply:
The version should probably be hardcoded here instead https://github.com/microsoft/AL-Go/blob/main/Actions/Packages.json (Like we did for the sign action)